src/input/readmo.F90

```diff
@@ -720,13 +720,13 @@ subroutine readmo
         end if
       end if
       maxci = 16000          
-      write (iw, '(1X,a)')"** Cite this program as: MOPAC, Version: "//verson//", James J. P. Stewart,     **"
-      write (iw, '(1X,a, a,a)')"**                           web-site: http://openmopac.net                  **"
-      write (iw, '(1X,a)')"*******************************************************************************"
       write (iw, '(1X,a)')"**                                                                           **"
-      write (iw,"(1x,a)") "**                                   MOPAC                                   **"
+      write (iw,"(1x,a)") "**                              MOPAC v"//verson//"                  **"
       write (iw, '(1X,a)')"**                                                                           **"
       write (iw, '(1X,a)')"*******************************************************************************"
+      write (iw, '(1X,a)')"**         Digital Object Identifier (DOI): 10.5281/zenodo.??????            **"

```

Comment:
Might be better to have this fed in by CMake as a macro?

Reply:
The current plan is to use the "concept DOI" that Zenodo will provide, which is a collective reference to all MOPAC versions hosted on Zenodo, in contrast with a "version DOI" that will be specific to each version. In this case, it won't change and can be hard-coded. If we switch to version DOI's, I will have to feed it in through CMake, which will complicate the packaging process - the automated Zenodo web hooks can take a few days to auto-detect new versions, so we may have to wait a few days to update a version-specific DOI in the code with each new release.